### ISSUE TITLE
Collapsible Sections

### ISSUE THREAD

**Issue by MotoMoto1234** [mouse0270/kasper#5]
Add the ability to collapse groups for cleaner navigation

**Comment by mouse0270** [comment]
This sounds simple but I have no idea on how to accomplish it. Like technically its fairly simple to accomplish but I really don't want to add another button, as I feel like the UI is already fairly cluttered. I could make it something like double tap / long press, but that wont be intuitive to the user to collapse. Also not really sure how `contenteditable` elements interact with click or touch events.

I am not opposed to the idea, just not sure the best way to make it both user friendly and accessible to users for functionality. 

TL;DR I am not saying no to this idea, just might not be in the next version unless I come up with a clever design way to add it that feels easy for the user to understand.
